src/main/java/com/farao_community/farao/rao_failure_handler/RaoFailureHandlerListener.java

```diff
@@ -56,9 +56,9 @@ public void onMessage(final Message message) {
             LOGGER.info("Failed RAO request received from DLQ: {}", raoRequest);
             addMetaDataToLogsModelContext(raoRequest.getId(), brokerCorrelationId, message.getMessageProperties().getAppId(), raoRequest.getEventPrefix());
             businessLogger.error("The RAO request reached the maximum number of attempts but could not be processed successfully");
-            sendErrorResponse("The RAO request reached the maximum number of attempts but could not be processed successfully", replyTo, brokerCorrelationId);
+            sendRaoFailedResponse(raoRequest.getId(), "The RAO request reached the maximum number of attempts but could not be processed successfully", replyTo, brokerCorrelationId);
         } catch (Exception e) {
-            sendErrorResponse("Unhandled exception: " + e.getMessage(), replyTo, brokerCorrelationId);
+            sendRaoFailedResponse("defaultId", "Unhandled exception: " + e.getMessage(), replyTo, brokerCorrelationId);
```

Comment:
I didn't like putting a "defautId" string as the value for the id here !

Reply:
After discussion with the team, we accept this version and an improvement can be made later.